ips/tristan.core.json

```diff
@@ -73,59 +73,59 @@
         "Description": "",
         "Project": "TRISTAN",
         "WI": "WI2.1.1",
-        "Partners": "ETH",
+        "Partners": "ETH Zürich",
         "Comment": ""
     },
     {
         "Repository": "UVM env for RISC-V verif",
         "URL": "https://github.com/openhwgroup/core-v-verif",
         "License": "Solderpad",
-        "Status": "In progress",
-        "Description": "",
+        "Status": "Finished, as the CV32A60X configuration of the CVA6 has reached TRL-5",
```

Comment:
Shall we just say Finished here, and add those other details into the description?

Reply:
Also, sometime I see "Released" some time I see "Finished", what do you prefer?